go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/segmentio/encoding
 
 go 1.13
+
+require github.com/dvyukov/go-fuzz-corpus v0.0.0-20190920191254-c42c1b2914c7
```

Comment:
go mod tidy pulled this in. should I remove it?

Reply:
Yes please :)